templates/components/listings/publications/tr_card.html.twig

```diff
@@ -0,0 +1,28 @@
+<div class="tr-list__item">
+    <div class="tr-list__item__header">
+        <h3>
+            <a href="{{ spec.latestVersionUri }}">{{ spec.title }}</a>
+        </h3>
+        {% block maturity %}{% endblock %}
+    </div>
+    <p><time datetime="{{ spec.date|date('Y-m-d') }}">{{ spec.date|format_date('long', locale='en-GB') }}</time>{% block history %} - <a href="{{ path('history_view', { 'spec': spec.specification.shortname }) }}">history</a>{% endblock history%}</p>
```

Comment:
To have consistent date formatting I believe that the following could become a macro:
```html
<time datetime="{{ spec.date|date('Y-m-d') }}">{{ spec.date|format_date('long', locale='en-GB') }}</time>
```

This woudl be similar to the `event_date` macro that @jean-gui developed for formatting events date in the main application (see: templates/macro.html.twig)

We could have another macro to display time

@jean-gui WDYT?

Reply:
shouldn't this be another component instead?

Reply:
Actually, I have deleted this event_date macro in the branch I'm currently working and replaced it with a twig function named event_range. But as it's name now suggests it's meant for ranges, not a single date.
We could have another twig function for displaying a single date with a time element around it but we certainly do not want to re-implement format_date itself (dependency injection would be helpful there).
I don't think a component for this would be very useful, if by component you mean an embeddable template.